package.json

```diff
@@ -41,7 +41,7 @@
     "dot-prop": "^4.2.0",
     "duplexify": "^3.5.1",
     "extend": "^3.0.1",
-    "grpc": "~1.9.1",
+    "grpc": "~1.10.0",
```

Comment:
@alexander-fenster Do we still want only 1.10.x, or can we go back to getting all 1.x updates?

Reply:
I'm ok with trying `"^1.10.0"`.